### PR TITLE
Include prevalence in login page detection

### PR DESCRIPTION
**Pull request by jkamdjou**

### CODE REVIEW

File: detection-rules/attachment_html_attachment_login_page.yml [hunk 1]
```diff
@@ -69,7 +69,10 @@ source: |
           )
   )
   and (
-    not profile.by_sender().solicited
+    (
+      not profile.by_sender().solicited
+      and not profile.by_sender().prevalence in ("new", "outlier")
```
Comment (rw-access) [inline detection-rules/attachment_html_attachment_login_page.yml:74]: This right? Has to be unsolicited _and old_?

Comment (jkamdjou) [reply detection-rules/attachment_html_attachment_login_page.yml:74]: right

Comment (morriscode) [reply detection-rules/attachment_html_attachment_login_page.yml:74]: so this won't ever fire on a new sender, the sender basically has to be common and unsolicited? I'm not sure i'm following the logic.

Comment (jkamdjou) [reply detection-rules/attachment_html_attachment_login_page.yml:74]: i was clearly sleep deprived when i wrote this. updating

Comment (jkamdjou) [inline detection-rules/attachment_html_attachment_login_page.yml:74]:
```suggestion
      and profile.by_sender().prevalence in ("new", "outlier")
```

**Review by morriscode — APPROVED** [review]
✔️

**Pull request merged**